app/src/main/java/mx/yellowme/youst/challenges/navigation/DeepLinkDestinationFragment.kt

```diff
@@ -8,8 +8,11 @@ import androidx.fragment.app.Fragment
 import mx.yellowme.youst.R
 
 class DeepLinkDestinationFragment : Fragment() {
-
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
I think this one can stay in the same line. It doesn't seem to reach the max line length 🤔